src/fragments/lib/storage/ios/list.mdx

```diff
@@ -1,7 +1,50 @@
 You can list all of the objects uploaded under a given prefix by setting the `pageSize`. If the `pageSize` is set lower than the total file size available, A single `Storage.list` call only returns a subset of all the files. To list all the files with multiple calls, the user can use the `nextToken` from the previous call response.
 
-This will list all public files:
+This will list all files in the path:
+<BlockSwitcher>
+
+<Block name="Async/Await">
+
+```swift
+let options = StorageListRequest.Options(pageSize: 1000)
+let listResult = try await Amplify.Storage.list(
+    path: .fromString("example/path"), 
+    options: options
+)
+listResult.items.forEach { item in
+    print("Path: \(item.path)")
+}
+```
+
+</Block>
+
+<Block name="Combine">
+
+```swift
+let sink = Amplify.Publisher.create {
+    let options = StorageListRequest.Options(pageSize: 1000)
+    try await Amplify.Storage.list(
+        path: .fromString("example/path"), 
+        options: options
+    )
+}.sink {
+    if case let .failure(error) = $0 {
+        print("Failed: \(error)")
+    }
+}
+receiveValue: { listResult in
+    print("Completed")
+    listResult.items.forEach { item in
+        print("Path: \(item.path)")
+    }
+}
+```
+
+</Block>
+
+</BlockSwitcher>
 
+This will list all public files (This behavior is deprecated):
```

Comment:
I would just remove this line and add `#### With Key (Deprecated)` like used elsewhere.

Reply:
Updated per suggestion.